collector/meminfo_numa_linux.go

```diff
@@ -33,8 +33,13 @@ const (
 	memInfoNumaSubsystem = "memory_numa"
 )
 
-type meminfoKey struct {
-	metricName, numaNode string
+var nodeRE = regexp.MustCompile(`.*devices/system/node/node([0-9]*)`)
```

Comment:
Sure, or I can move it into a smaller scope if there's a reasonable way to do this without having to recompute it all the time.